.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "plugins": [
+  "prettier"
```

Comment:
Could you check indentation of sub-section : `plugins` and `extends` ?  (add spaces)
Strapi use single quote in  his JSON config file, replace `"` by `'` (like https://github.com/strapi/strapi/blob/master/.eslintrc.back.js)

Reply:
This difference occurs because I'm using `.json` file and strapi project uses `.js` file. What do you prefer? I can change it.

Reply:
In fact :) My mistake ! `.json` is great

Reply:
Indentation solved.